code/modules/mob_spawn/ghost_roles/unused_roles.dm

```diff
@@ -307,10 +314,18 @@
 
 /datum/outfit/syndicatespace/syndicaptain
 	name = "Syndicate Ship Captain"
+	id = /obj/item/card/id/advanced/black/syndicate_command/captain_id
 	uniform = /obj/item/clothing/under/syndicate/combat
 	suit = /obj/item/clothing/suit/armor/vest/capcarapace/syndicate
+<<<<<<< HEAD
 	head = /obj/item/clothing/head/hos/beret/syndicate
+=======
+>>>>>>> 581828c4211 (Reorganize the outfits under 'mob_spawn' (#72345))
 	ears = /obj/item/radio/headset/syndicate/alt/leader
+	head = /obj/item/clothing/head/hats/hos/beret/syndicate
```

Comment:
```suggestion
	ears = /obj/item/radio/headset/syndicate/alt/leader
	head = /obj/item/clothing/head/hats/hos/beret/syndicate
```
PR needs updating to Master to account for #18191; this has been changed to head/hats/hos